snapcraft/internal/build_providers/_lxd/_images.py

```diff
@@ -26,11 +26,19 @@ def get_image_source(*, base: str) -> Dict[str, str]:
     :returns: a dictionary with a valid source for PyLXD.
     :raises KeyError: when a non supported base is used.
     """
+    
```

Comment:
why not treat the exception in lxd and switch to the devel stream? Makes it future proof. I do not recommend this change though as we are moving to the same images used for launchpad builds moving forward.

Reply:
I accidently removed the history.

The change was:

```
-_BASE_IMAGE = dict(core="16.04", core16="16.04", core18="18.04")                                                                                                                                                                            
+_BASE_IMAGE = dict(core="16.04", core16="16.04", core18="18.04", core20="20.04")
```